spec/struct.dd

```diff
@@ -1106,6 +1106,26 @@ $(H3 $(LNAME2 field-init, Field initialization inside a constructor))
 
 $(H2 $(LEGACY_LNAME2 StructCopyConstructor, struct-copy-constructor, Struct Copy Constructors))
 
+    $(P $(RED Warning:) The plan is for
+    $(RELATIVE_LINK2 struct-copy-constructor, copy constructors) to replace
+    $(RELATIVE_LINK2 struct-postblit, postblit constructors), leaving postblit
+    constructors only in legacy code. However, because the compiler hooks in
+    druntime which deal with dynamic arrays and associative arrays have not yet
+    all been updated to support copy constructors properly
+    ($(HTTPS github.com/dlang/dmd/issues/20970, issue #20970)), any type that might
+    be used in a dynamic array or associative array should use a postblit
+    constructor rather than a copy constructor. The copy constructor will not
+    be called in all the cases where it should be for the elements of a dynamic
+    array or the keys or values in an associative array. Postblit constructors
+    do not have this problem.)
+
+    For backward compatibility reasons, a `struct` that explicitly defines both
+    a copy constructor and a postblit will only use the postblit for implicit
+    copying. However, if the postblit is disabled, the copy constructor will be
+    used. If a struct defines a copy constructor (user-defined or generated)
+    and has fields that define postblits, a deprecation will be issued,
+    informing that the postblit will have priority over the copy constructor.)
```

Comment:
This closing `)` has no opening `$(P` tag. Ditto for the postblit 2nd paragraph.

Reply:
As the warnings are each 2 paragraphs, might be nice to wrap them in a `$(PANEL`.

Reply:
Nice catch. I clearly screwed it up when I was rearranging the text and then failed to catch it. I really should just take the time to figure out how to build the documentation locally again to see what it actually looks like. It's been ages since I've done that and didn't want to bother, but I'm clearly making too many mistakes here which would be easily caught if I actually built the documentation.

And I'll have a look at what `$(PANEL` does. I don't recall seeing that particular macro before.

Reply:
Okay. I finally figured out how to build the documentation and see the changes. It looks fine now. I don't know if using PANEL looks better or not, since it's grouping the two paragraphs, but it doesn't affect the count (though the count is kind of odd in a few places already anyway), so it looks a bit funny in that regard. But it probably is better to group them, so the changes now use it.